### ISSUE TITLE
future: do you need a Chinese document?

### ISSUE THREAD

**Issue by RyanHo97** [prabhuignoto/react-chrono-docs#4]
Do you need a Chinese document? I came across it by chance and may use your component to create a website in the future. I would like to help you write a version of the Chinese document

**Comment by prabhuignoto** [comment]
@RyanHo97 Thank you for dropping by, and I'm pleased to hear that you're contemplating using the component. Absolutely, having a Chinese document would be fantastic. Please feel free to open a PR and let me know if you need help

**Comment by RyanHo97** [comment]
Great, I'm planning to supplement Chinese documents in my future spare time. I see that you are using vitepress to build a website. Do you want to use i18n to distinguish between en/zh document paths.

**Comment by prabhuignoto** [comment]
@RyanHo97 yes vitepress is used and sure we can use the [i18n](https://vitepress.dev/guide/i18n)

**Comment by RyanHo97** [comment]
Requesting help me, I have not found the best practices for using i18n with vitepress. If you can find them, please let me know or create the/zh directory directly in the directory. Thank you

**Comment by RyanHo97** [comment]
The vite document has different repositories for different language versions
![image](https://user-images.githubusercontent.com/49436413/236483232-a1f1b95b-b98f-45b6-ad0d-91d1e28d1985.png)
